### ISSUE TITLE
Yaml converts with tabs, that cannot be formatted by prettier

### ISSUE THREAD

**Issue by sergeysova** [lamartire/sharec#156]
And do not supported by github

<details>
<summary>Before</summary>

```yaml
name: Node Library Config (Branch CI)

on:
  pull_request:
  push:
    branches:
      - main
      - master
      - develop

jobs:
  build:
    name: Build Project
    runs-on: ubuntu-20.04
    strategy:
      matrix:
        node-version: [12.x]
    steps:
      - name: Checkout code
        uses: actions/checkout@v2

      - name: Use Node.js ${{ matrix.node-version }}
        uses: actions/setup-node@v1
        with:
          node-version: ${{ matrix.node-version }}

      - run: rm -rf .npmrc
      - run: echo "//npm.pkg.github.com/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
      - run: npm install
      - run: npm run test
      - run: npm run build
      - run: npm publish
```

</details>

<details>
<summary>After</summary>

```yaml
name: Node Library Config (Branch CI)
"on":
	pull_request: null
	push:
		branches:
			- main
			- master
			- develop
jobs:
	build:
		name: Build Project
		runs-on: ubuntu-20.04
		strategy:
			matrix:
				node-version:
					- 12.x
		steps:
			- name: Checkout code
				uses: actions/checkout@v2
			- name: "Use Node.js ${{ matrix.node-version }}"
				uses: actions/setup-node@v1
				with:
					node-version: "${{ matrix.node-version }}"
			- run: rm -rf .npmrc
			- run: >-
					echo "//npm.pkg.github.com/:_authToken=${{ secrets.NPM_TOKEN }}" >
					~/.npmrc
			- run: npm install
			- run: npm run test
			- run: npm run build
			- run: npm publish
```

</details>

[Prettier](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEUCGBbOSAEA5CAEzhwBkBLAIwCc1qBPHAYWgDNyBzHACgCFaoYABbMAkgEoAOlEkhospNMkwADgFcANhoD61OAEc1cAM4xcUTRqWq1xoYpkxlNNIKEmHyrzAC0ODGjkMo7efgGmcNTWoTjEAG5wGhAq0gBWEJTGnjCUauQahNnK6Fi4vHkFOAAK1BCpcGBOITDUalDGPtC4armwaj4ATAAMAHRDACzRMKa08Bz0RU4wAS3kAB6L3lBEcD4J1MbkXVPeyn4AjAMja1MRKlknvjgl2MzuYADWEGowOJDEj2Utg8ODQjSO7QA9MIGl8fgABOIDR5+F64WQAVWMJAIxBGqWMOAAJMBgP40Ks1iNtsQ9pFDtAcABfJmyQEwYFZUHg6DGSHYjkqHw0uCI87sgDu5Bg9nZxR2dIOEPRIBJZJW1HW1IV+wZUGZrJAKJwrSguGoGBwPmorBw1JUGGoYGNptwAD4fHL4MIIDhZJDIVAHSMVB8OCMONKhD0RpAMJCkNo0D8hAAVCAfBAAXjVOGxYD00xGeCqAFltKmAPIAaQAongDbIcG6vQA-SH2x3O5pnE1tcwOnBBUxoLQu-vPQemnDwUzjs2Ty3T3L5QjzgeW9SUDTkOwgAA0chUMAhxmQoDotQlVToCDPKFHErQ9DPh5cnzgMAAyiowUEOMgLRGIecAYJQcCEMQhCkK4HBqGgHBwAAYhAFoUieUAAQ+PwQAeIBCMsGgAOpCNKJi-mAcBfne0rkHE0r0Mg4DGK+IDDpEMA1AhATIKwo7YoeBJrPwYKZt+mBwBQUBwLx-FwIJxhrF+-4aHAACKagQPAskaAJIC-gckRMfQmAaHhKiarARHkIQMrIAAHEMh4WRA2JEbQKhMRZJiRAkeGGFpcA1Mk94gGgHTSRBEF4XohjkHoXEcDxSB8bp8kgNiGDkIBrTpYcmGqRpgU6XpMBoJQ1m2UIyADIeLSBDumEsBgyUgCYACseHAqm5X3qlelxEYohQMQsBfgW5DHgAgiNX4wPQqklXALJAA)

**Comment by sanderv1992** [comment]
Not Working Well, it will be very helpful, if there will be an option for copy -> paste files, without merging and formatting.

**Comment by lamartire** [comment]
Hello guys. Will check it tomorrow!

**Comment by sanderv1992** [comment]
Thank you @lamartire

**Comment by lamartire** [comment]
@sanderv1992, @sergeysova hot fix available in `2.5.2`. Sometimes later will fix it much more complexly.

**Comment by sergeysova** [comment]
Thank you!

**Comment by lamartire** [comment]
Fix it again in `2.5.3`. Do not forget to update, please @sergeysova, @sanderv1992!
